api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -9280,5 +9280,23 @@
 			<column name="sort_weight" type="double"></column>
 		</addColumn>
 	</changeSet>
+
+
+	<changeSet id="201604030003" author="vishnuraom">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<sqlCheck expectedResult="0">SELECT count(*) FROM drug_order WHERE dosing_type = 'SIMPLE' OR dosing_type = 'FREE_TEXT';</sqlCheck>
+			</not>
+		</preConditions>
+		<comment>Converting values in drug_order.dosing_type column(TRUNK-4845)</comment>
+		<update tableName="drug_order">
```

Comment:
You should never  combine changesets like this because:
1. The precondition tests 2 scenarios where some installations can meet just one of them
2. If one of the updates fails, it should not affect the other.